src/services/post-install-utility.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-console */
+const chalk = require('chalk');
+const { Config } = require('@twilio/cli-core').services.config;
+const { configureEnv } = require('@twilio/cli-core');
+
+const PORT_WARNING = `Profiles exist with API keys in system keychain. Please run ${chalk.bold(
+  'twilio profiles:port',
+)} to port keys to config.`;
```

Comment:
update to - "port all keys to the config file"?

Reply:
Done!